src/CoreText/CTFontCollection.cs

```diff
@@ -178,6 +178,22 @@ public CTFontDescriptor[] GetMatchingFontDescriptors ()
 			return matches;
 		}
 
+		[Mac (10,7), iOS (12,0), TV (12,0), Watch (5,0)]
+		[DllImport (Constants.CoreTextLibrary)]
+		static extern IntPtr CTFontCollectionCreateMatchingFontDescriptorsWithOptions (IntPtr collection, IntPtr options);
+
+		[Mac (10,7), iOS (12,0), TV (12,0), Watch (5,0)]
+		public CTFontDescriptor [] GetMatchingFontDescriptors (CTFontCollectionOptions options)
```

Comment:
These are manual bindings, so they should have manual tests too.

Reply:
@rolfbjarne can we run the tests on the old-macOS from the PR ?

Reply:
@spouliot no, but what I can do is create a new PR that has both the changes from this PR and #4278.

Reply:
k, will catch it at merge time then :)